tags.json

```diff
@@ -0,0 +1,49 @@
+{
+    "advancement": "Any change to the technical advancement system",
+    "api": "Common API changes, such as updates URLs",
+    "assets": "Asset changes, such as textures, sounds, or lang",
+    "attribute": "Changes to the attribute registry",
+    "block": "New blocks or changes to block data",
+    "block-entity": "Used alongside `block` to specify block entities",
+    "blockstate": "Changes to blockstates",
+    "breaking": "Any change that will break/cause errors with existing packs",
+    "chat-type": "Anything regarding chat types",
+    "command": "New commands or changes to command syntax/behavior",
+    "damage-type": "Changing any damage type or tag",
+    "data": "Any changes to data structure; an overreach of `tag`, `block`, and occasionally `worldgen`",
```

Comment:
Data is supposed to mean changes to the vanilla data. Not changes to the format/structure/codecs.